docs/user/integration/sap-cloud-logging/README.md

```diff
@@ -1,57 +1,57 @@
 # Integrate with SAP Cloud Logging
 
-## Overview
-
 | Category| |
 | - | - |
 | Signal types | logs, traces, metrics |
 | Backend type | third-party remote |
 | OTLP-native | yes for traces and metrics, no for logs |
 
-Learn how to define LogPipelines and TracePipelines to ingest application and access logs as well as distributed trace data in instances of [SAP Cloud Logging](https://help.sap.com/docs/cloud-logging?locale=en-US&version=Cloud).
-
-SAP Cloud Logging is an instance-based and environment-agnostic observability service that builds upon OpenSearch to store, visualize, and analyze logs, metrics, and traces.
-
-![setup](./../assets/sap-cloud-logging.drawio.svg)
+Learn how to configure the Telemetry module to ingest application and access logs as well as distributed trace data and metrics in instances of SAP Cloud Logging.
 
 ## Table of Content
 
 - [Integrate with SAP Cloud Logging](#integrate-with-sap-cloud-logging)
-  - [Overview](#overview)
   - [Table of Content](#table-of-content)
   - [Prerequisites](#prerequisites)
+  - [Overview](#overview)
   - [Ship Logs to SAP Cloud Logging](#ship-logs-to-sap-cloud-logging)
     - [Set Up Application Logs](#set-up-application-logs)
     - [Set Up Access Logs](#set-up-access-logs)
   - [Ship Distributed Traces to SAP Cloud Logging](#ship-distributed-traces-to-sap-cloud-logging)
   - [Ship Metrics to SAP Cloud Logging](#ship-metrics-to-sap-cloud-logging)
-  - [Kyma Dashboard Integration](#kyma-dashboard-integration)
-  - [SAP Cloud Logging Alerts](#sap-cloud-logging-alerts)
-    - [Import Alerts](#import-alerts)
-    - [Recommended Alerts](#recommended-alerts)
-  - [SAP Cloud Logging Dashboards](#sap-cloud-logging-dashboards)
+  - [Set Up Kyma Dashboard Integration](#set-up-kyma-dashboard-integration)
+  - [Use SAP Cloud Logging Alerts](#use-sap-cloud-logging-alerts)
+  - [Use SAP Cloud Logging Dashboards](#use-sap-cloud-logging-dashboards)
 
 ## Prerequisites
 
-- Kyma as the target deployment environment.
-- The [Telemetry module](../../README.md) is [added](https://kyma-project.io/#/02-get-started/01-quick-install).
+- SAP BTP, Kyma runtime as the target deployment environment.
```

Comment:
Tricky.. Can't we express it different? We are in the OSS docu here, we cannot say it is a prerequisute to use the SAP product.
At the end you need a kubernetes cluster with the telemetry module installed/enabled..

Reply:
This is the CLS guide. Pls confirm, CLS works with an OS cluster?

Reply:
yes, the telemetry module can be used on any kubernetes cluster and will enable you to integrate with CLS